tests/test_morphsqueeze.py

```diff
@@ -170,3 +170,141 @@ def test_morphsqueeze_extrapolate(user_filesystem, squeeze_coeffs, wmsg_gen):
     )
     with pytest.warns(UserWarning, match=expected_wmsg):
         single_morph(parser, opts, pargs, stdout_flag=False)
+
+
+@pytest.mark.parametrize(
+    "squeeze_coeffs, x_morph",
+    [
+        ({"a0": 0.01, "a1": 0.01, "a2": -0.1}, np.linspace(0, 10, 101)),
+    ],
+)
+def test_non_strictly_increasing_squeeze(squeeze_coeffs, x_morph):
+    x_target = x_morph
+    y_target = np.sin(x_target)
+    coeffs = [squeeze_coeffs[f"a{i}"] for i in range(len(squeeze_coeffs))]
+    squeeze_polynomial = Polynomial(coeffs)
+    x_squeezed = x_morph + squeeze_polynomial(x_morph)
+    # non-strictly-increasing
+    assert not np.all(np.sign(np.diff(x_squeezed)) > 0)
+    y_morph = np.sin(x_squeezed)
+    # all zero initial guess
+    morph_results = morphpy.morph_arrays(
+        np.array([x_morph, y_morph]).T,
+        np.array([x_target, y_target]).T,
+        squeeze=[0, 0, 0],
+        apply=True,
+    )
+    _, y_morph_actual = morph_results[1].T  # noqa: F841
+    y_morph_expected = np.sin(x_morph)  # noqa: F841
+    # squeeze morph extrapolates.
+    #   Need to extract extrap_index from morph_results to examine
+    #   the convergence.
+    # assert np.allclose(y_morph_actual, y_morph_expected, atol=1e-3)
+    # Raise warning when called without --check-increase
+    with pytest.warns() as w:
+        morph_results = morphpy.morph_arrays(
+            np.array([x_morph, y_morph]).T,
+            np.array([x_target, y_target]).T,
+            squeeze=[0.01, 0.01, -0.1],
+            apply=True,
+        )
+    assert w[0].category is UserWarning
+    actual_wmsg = " ".join([str(w[i].message) for i in range(len(w))])
+    expected_wmsg = (
+        "Warning: The squeeze morph has interpolated your morphed "
```

Comment:
Let's make this less scary and more helpful to the user.  How about something like:
```After morphing your x-grid is no longer monotonically increasing.  This is allowed, but please sanity-check your results.  If you like, you may rerun with a less flexible morph (e.g., lower order polynomial) to address this.
To force the code to error when this occurs, please rerun with the option --force-monotonic
```